Cue/app/tabs/library/DeckViewInfoHeader.js

```diff
@@ -78,7 +78,9 @@ class DeckViewInfoHeader extends React.Component {
 
       subtitleText = this.props.deck.owner === this.props.userId
             ? " by you."
-            : " by someone else."
+            : (this.props.deck.author === null
```

Comment:
I would reverse this check to display the author name if the value is truthy, i.e., `this.props.deck.author ? < by author > : < by someone else >`.